internal/aliasfix/main.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build go1.18
+// +build go1.18
+
+package main
+
+import (
+	"bytes"
+	"flag"
+	"go/ast"
+	"go/format"
+	"go/parser"
+	"go/token"
+	"io"
+	"io/fs"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"golang.org/x/tools/imports"
+)
+
+var (
+	fset = token.NewFileSet()
+)
+
+func main() {
+	flag.Parse()
+	path := flag.Arg(0)
+	if err := processPath(path); err != nil {
+		log.Fatal(err)
+	}
+}
+
+func processPath(path string) error {
+	dir, err := os.Stat(path)
+	if err != nil {
+		return err
+	}
+	if dir.IsDir() {
+		filepath.WalkDir(path, func(path string, d fs.DirEntry, err error) error {
+			if err == nil && !d.IsDir() && strings.HasSuffix(d.Name(), ".go") {
+				err = processFile(path, nil)
+			}
+			if err != nil {
+				return err
+			}
+			return nil
+		})
+	} else {
+		if err := processFile(path, nil); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func processFile(name string, w io.Writer) error {
```

Comment:
Nit: Can you add a comment that `w io.Writer` is present in the signature for test accessibility ? I was wondering to myself why it was there until I got to the test 😅

Reply:
done.